### ISSUE TITLE
ModuleNotFoundError: No module named 'pyvex.vex_ffi'

### ISSUE THREAD

**Issue by blu3sh0rk** [angr/pyvex#285]
Hello,
I met some problem when i `import pyvex`.

```
(angr) lucas@rex:~/Desktop/rex/pyvex$ ipython
Python 3.8.10 (default, Jun 22 2022, 20:18:18) 
Type 'copyright', 'credits' or 'license' for more information
IPython 8.5.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import pyvex
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Cell In [1], line 1
----> 1 import pyvex

File ~/Desktop/rex/pyvex/pyvex/__init__.py:17
     15 from typing import Any
     16 import cffi
---> 17 from .vex_ffi import ffi_str as _ffi_str
     18 ffi = cffi.FFI()
     20 import logging

ModuleNotFoundError: No module named 'pyvex.vex_ffi'
```

**Comment by rhelmot** [comment]
Have you built the package? vex_ffi is autogenerated at build time.

**Comment by twizmwazin** [comment]
If you're installing from source using -e (if using angr-dev, you are), you either need to downgrade setuptools to 64.0.1 or checkout https://github.com/angr/pyvex/pull/283. See https://github.com/angr/angr/issues/3487 for more details

**Comment by twizmwazin** [comment]
Fixed as of 1746704701123912ccc69742aa9d6168b3475738. CI and angr-dev still use an older setuptools, but pyvex should now work correctly with newer setuptools as well.
